presto-native-execution/presto_cpp/main/PrestoServerOperations.cpp

```diff
@@ -143,7 +143,7 @@ std::string PrestoServerOperations::systemConfigOperation(
       return fmt::format(
           "{}\n",
           SystemConfig::instance()->optionalProperty(name).value_or(
-              "<default>"));
+              fmt::format("Could not find property '{}'", name)));
```

Comment:
Can we make sure to return HTTP 500 on errors?

Reply:
Yeah we can do that